Look in the MacPorts Applications dir before system and user Applications dirs.
--- pashua.sh.orig	2015-02-23 08:01:43.000000000 -0600
+++ pashua.sh	2020-06-19 15:52:54.000000000 -0500
@@ -19,8 +19,9 @@
     searchpaths[1]="$mypath/Pashua"
     searchpaths[2]="$mypath/$bundlepath"
     searchpaths[3]="./$bundlepath"
-    searchpaths[4]="/Applications/$bundlepath"
-    searchpaths[5]="$HOME/Applications/$bundlepath"
+    searchpaths[4]="@APPLICATIONS_DIR@/$bundlepath"
+    searchpaths[5]="/Applications/$bundlepath"
+    searchpaths[6]="$HOME/Applications/$bundlepath"
 
     for searchpath in "${searchpaths[@]}"
     do
